weavertest/deployer.go

```diff
@@ -310,9 +310,11 @@ func (d *deployer) startGroup(group *group) error {
 		group.envelopes = append(group.envelopes, e)
 
 		// Run the envelope.
-		d.stopped.Add(1)
+		//
+		// TODO(mwhittaker): We should add 'd.stopped.Add(1)' and 'defer
```

Comment:
@spetrovic77, I ran into this bug. The changes you're making to remove restarts from `Envelope` might make it easier to fix this bug. If you're interested in fixing, go ahead! Otherwise, I can pick it up :)

Reply:
Ack.